### PR TITLE
Cleanup driver

### PR DESCRIPTION
**Pull request by applmak**

### CODE REVIEW

File: server/modules/playlist_driver.js [hunk 1]
```diff
@@ -15,151 +15,181 @@ limitations under the License.
...
-
-      // Now, in so many seconds, we'll need to switch to another module 
-      // or another layout. How much time do we have?
-      if (newLayoutTime < newModuleTime) {
-        timer = setTimeout(() => ret.nextLayout(), time.until(newLayoutTime));
-      } else {
-        timer = setTimeout(() => ret.nextModule(), time.until(newModuleTime));
-      }
+class PlaylistDriver {
+  static makeDriver(moduleSM) {
```
Comment (bmt) [inline server/modules/playlist_driver.js]: Why not just use the constructor?

File: server/modules/playlist_driver.js [hunk 2]
```diff
@@ -15,151 +15,181 @@ limitations under the License.
...
+    if (monitor.isEnabled()) {
+      monitor.update({playlist: {
+        time: time.now(),
+        event: `change module ${module}`,
+        deadline: this.getNextDeadline(),
+      }});
+    }
+
+    // Now, in so many seconds, we'll need to switch to another module 
+    // or another layout. How much time do we have?
```
Comment (bmt) [inline server/modules/playlist_driver.js]: I see "layout" mentioned all over in the code and comments. Weren't you working on removing that concept?

File: server/modules/playlist_driver.js (hunk 1 above)
Comment (applmak) [reply server/modules/playlist_driver.js]: This is a static constructor.

Honestly, I think I was going to put the error-listener assignment here, but I didn't. Would you like me to do that here?

File: package.json [hunk 3]
```diff
@@ -1,27 +1,28 @@
 {
   "name": "chicago-brick",
-  "version": "0.3.7",
+  "version": "0.4.0",
```
Comment (bmt) [inline package.json]: This is already at head. Is there a way to clean this up?

File: server/modules/playlist_driver.js (hunk 1 above)
Comment (bmt) [reply server/modules/playlist_driver.js]: I'd either put that in there or just drop the static constructor all together and change the caller to ```new PlaylistDriver(moduleSM)```.

Comment (applmak) [reply server/modules/playlist_driver.js]: Sure.

File: package.json (hunk 3 above)
Comment (applmak) [reply package.json]: Yep.

**Pull request merged**